.setup/bin/setup_sample_courses.py

```diff
@@ -1561,6 +1562,15 @@ def create(self, conn, gradeable_table, electronic_table, reg_table, component_t
 
 
 
+        if self.peer_grading == True:
+            with open(os.path.join(SETUP_DATA_PATH, 'random', 'graders.txt')) as graders, \
+            open(os.path.join(SETUP_DATA_PATH, 'random', 'students.txt')) as students:
+                graders = graders.read().strip().split()
+                students = students.read().strip().split()
+                length=len(graders)
+                for i in range(length):
+                    conn.execute(peer_assign.insert(), g_id=self.id, grader_id=graders[i], user_id=students[i])
+            
```

Comment:
The `Course` class contains a `users` field which is a list of all students and staff for a course. These users should have the information about user type/section needed to perform filtering. At the moment, the `Gradeable` class does not have access to the corresponding course object, but it would be easy to pass that in. Then, we could get rid of `random/students.txt` and `random/graders.txt` and instead use `course.users` and the same randomization algorithm (by offsets) used on the `php` site to perform peer matchings.

Reply:
Thanks! Should I update on this PR or as we were discussing yesterday - incorporate changes including randomization in "Sample Courses -2" PR? @emaicus